### ISSUE TITLE
can I run Sourcetrail with openjdk11?

### ISSUE THREAD

**Issue by silloy** [CoatiSoftware/Sourcetrail#1028]
I hava download sourcetrail, but I install jdk11,can't run? 

![image](https://user-images.githubusercontent.com/12909107/83251193-d9128200-a1db-11ea-9384-009e6ce18133.png)

**Comment by silloy** [comment]
my version 2020.1.117

**Comment by silloy** [comment]
when I revert to 2019.4.102, it run well?

**Comment by egraether** [comment]
Using a path of the form `/Library/Java/JavaVirtualMachines/<jdk_version>/Contents/MacOS/libjli.dylib` should work. Please see [this comment](https://github.com/CoatiSoftware/Sourcetrail/issues/955#issuecomment-637073264)

**Comment by egraether** [comment]
I'm closing this, please reopen if you still have issues.

**Comment by Etienne-Schmitt** [comment]
Hi,

I've done it myself but exactly the same problem!
I used OpenSDK 11 (11.0.7+10) and 8 (1.8.0_252-b09) with the exact path you gave

![image](https://user-images.githubusercontent.com/22135894/83664366-2de23e00-a5ca-11ea-8a5c-03bdf96e7734.png)

Have a nice day

**Comment by silloy** [comment]
@Etienne-Schmitt  you can try revert version  to 2019.4.102, I has revert and it works well

**Comment by Etienne-Schmitt** [comment]
Yes ofc I already do it when I work but this is still an issue and you could reopen the issue this need to be investigated...

**Comment by Zwischenablage** [comment]
I have the exact same issue (JDK jdk1.8.0_261.jdk) and the latest Sourcetrail version Sourcetrail_2020_2_43_macOS_64bit.
I've also tried JDK 14.02, but no path I've provided within settings did the trick.


Everything works, as soon as I've reverted to 2019.4.61

**Comment by Downstream1998** [comment]
I wonder how I should configure JRE System Library if I rum Sourcetrail with JDK 9 or later?

In the later version of JDK, there is no jar files, they use jmod instead.

<img width="634" alt="无标题" src="https://user-images.githubusercontent.com/39040474/103597931-59e8e780-4f3c-11eb-91a4-8684af8ce98a.png">

**Comment by mlangkabel** [comment]
Good point. I think we need to handle this case at https://github.com/CoatiSoftware/Sourcetrail/blob/master/java_indexer/src/main/java/com/sourcetrail/JavaIndexer.java#L89

**Comment by mlangkabel** [comment]
I just checked but it looks like the `JRE System Library` setting is a relict form the past when Sourcetrail was based on JavaSymbolSolver. So I tested #418 again, without anything in the `JRE System Library` and it works! Maybe we can remove that setting now. 
@Downstream1998: Is everything working fine if you leave `JRE System Library` empty? Or do you experience issues?
